nautobot/extras/tests/test_models.py

```diff
@@ -809,6 +810,50 @@ def test_timeout_validation(self):
         ei.timeout = 65536
         ei.validated_save()
 
+    def test_render_extra_config(self):
+        ei_with_extra_config = ExternalIntegration.objects.filter(extra_config__isnull=False)
+        ei_without_extra_config = ExternalIntegration.objects.filter(extra_config__isnull=True)
+        self.assertEqual(
+            ei_with_extra_config.first().render_extra_config({}),
+            ei_with_extra_config.first().extra_config,
+        )
+        self.assertEqual(
+            ei_without_extra_config.first().render_extra_config({}),
+            {},
+        )
+        ei = ei_with_extra_config.first()
+        ei.extra_config = {"context": "{% foo %}"}
+        ei.save()
+        with self.assertRaises(TemplateSyntaxError):
+            ei.render_extra_config({})
+
+        ei.extra_config = "{{ data | notvalid }}"
+        ei.save()
+        with self.assertRaises(TemplateAssertionError):
+            ei.render_extra_config({})
```

Comment:
Am I missing something or do we not have any positive tests here (valid J2 template, data gets substituted as expected)?